src/Authentication/Helpers/AuthenticationHelper.cs

```diff
@@ -380,5 +382,11 @@ public static int GetUserId(HttpContext context)
 
             return 0;
         }
+
+        private static bool IsValidUriWithHttps(string uri)
+        {
+            return Uri.TryCreate(uri, UriKind.Absolute, out Uri? validatedUri)
```

Comment:
Don't parse a uri that's already been parsed.

Reply:
ok